### PR TITLE
Changing Commissioning 2020 to Commissioning 2021 with new run # and dataset. 

### PR DESCRIPTION
**Pull request by bbilin**
Addition to PR #34613 discussions

### CODE REVIEW

**Comment by chayanit** [comment]
please test

Hello @boudoul , do we need this PR urgently? 12_0_0 is closing now.

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -490,8 +490,9 @@
 steps['RunCosmics2015C']={'INPUT':InputInfo(dataSet='/Cosmics/Run2015C-v1/RAW',label='2015C',run=[256259],events=100000,location='STD')}
 steps['RunCosmics2016B']={'INPUT':InputInfo(dataSet='/Cosmics/Run2016B-v1/RAW',label='2016B',run=[272133],events=100000,location='STD')}
 
-### LS2 - MWGR#5 2020 - CSC, DAQ, DCS, DQM, DT, ECAL, GEM, HCAL, RPC, TCDS, TRACKER, TRG ###
-steps['RunCosmics2020']={'INPUT':InputInfo(dataSet='/ExpressCosmics/Commissioning2020-Express-v1/FEVT',label='2020GR5',ls={338714: [[1, 10000]]},events=100000,location='STD')}
+### LS2 - MWGR 2021 - CSC, DAQ, DCS, DQM, DT, ECAL, GEM, HCAL, RPC, TCDS, TRACKER, TRG ###
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_steps.py:493]: Should you update this commen also?

With 343498, we have
CSC, DAQ, DCS, DQM, DT, ECAL, GEM, HCAL, L1SCOUT, PIXEL, RPC, TCDS, TRACKER, TRG
Information can be found in
https://cmsoms.cern.ch/cms/runs/report?cms_run=343498&cms_run_sequence=GLOBAL-RUN

Comment (bbilin) [reply Configuration/PyReleaseValidation/python/relval_steps.py]: done. thanks for correcting

**Comment by srimanob** [comment]
Please test

**Comment by srimanob** [comment]
Please abort

**Comment by srimanob** [comment]
test parameters:
- workflows = 138.1,138.2

**Comment by srimanob** [comment]
Please test

**Comment by srimanob** [comment]
+Upgrade

This PR is to update the input of the commissioning workflows 138.1 and 138.2 from 2020 to 2021 data. New workflows run successfully.

**Comment by bbilin** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**